KLSadd_insertion/src/updateChapters.py

```diff
@@ -1,110 +1,367 @@
-Rahul Shah
-self-taught python don't kill me I know its bad
-2/5/16
-Re-write of linetest.py so people other than me can read it
-This project was/is being written to streamline the update process of the book used
-for the online repository, updated via the KLSadd addendum file which only affects chapters
-9 and 14
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-NOTE! AS OF 2/26/16 THIS FILE IS NOT YET UPDATED TO THE FULL CAPACITY OF THE PREVIOUS FILE. IF FURTHER DEVELOPMENT IS NEEDED, REFER TO THE
-linetest.py FILE IF THIS FILE DOES NOT ADDRESS THE NEW/EXTRA GOALS. This means that XCITE PARSE etc HAS NOT BEEN ADDED
-Additional goals (already addressed in linetest.py):
--insert correct usepackages
--insert correct commands found in KLSadd.tex to chapter files
--insert editor's initials into sections (ie %RS: begin addition, %RS: end addition)
-Additional goals (not already addressed in linetest.py):
--rewrite for "smarter" edits (ex. add new limit relations straight to the limit relations section in the section itself, not at the end)
-Current update status: incomplete
-Goals:change the book chapter files to include paragraphs from the addendum, and after that insert some edits so they are intelligently integrated into the chapter
-
-Edward Bian
-Currently under heavy modification; sections may not work and/or look inefficient/confusing
-"""
+# Path to data directory
+DATA_DIR = os.path.dirname(os.path.realpath(__file__)) + "/../data/"
+
+
 
 # start out by reading KLSadd.tex to get all of the paragraphs that must be added to the chapter files
 # also keep track of which chapter each one is in
 
-# variables: chapNums for the chapter number each section belongs to paras will hold the sections that will be copied over
+
 # chapNums is needed to know which file to open (9 or 14)
-# mathPeople is just the name for the sections that are used, like Wilson,
-# Racah, etc. I know some are not people, its just a var name
-chapNums = []
+# mathPeople is just the name for the sections that are used, like Wilson, Racah, etc.
+chap_nums = []
 paras = []
-klsparas = []
-mathPeople = []
-newCommands = []  # used to hold the indexes of the commands
-ref9II = []  # Hold section search indexes
-ref14II = []
-ref9III = []  # Holds all indexes
-ref14III = []
+
+# klsaddparas contains the locations of things with "paragraph{" in KLSadd, these are subsections that need to be sorted
+klsaddparas = []
+math_people = []
+new_commands = []  # used to hold the indexes of the commands
+
+# ref9II holds the beginnings and ends of each chapter 9 section, provided and area to search
+ref9_2 = []
+
+# ref14II holds the beginnings and ends of each chapter 14 section, provided and area to search
+ref14_2 = []
+
+# ref9III holds all subsections in each chapter 9 section along with the what  ref9II holds
+ref9_3 = []
+
+# ref14III holds all subsections in each chapter 14 section along with the what ref14II holds
+ref14_3 = []
```

Comment:
Discrepancies between comments and actual variable names - ref9_3 vs. ref9III. Also, I feel like these really shouldn't be their own variables, or at least not named like this.